src/lib.c

```diff
@@ -93,7 +103,7 @@ efi_append_variable(efi_guid_t guid, const char *name, const uint8_t *data,
 	int rc;
 	if (!ops->append_variable) {
 		rc = generic_append_variable(guid, name, data, data_size,
-					     attributes);
+						 attributes);
```

Comment:
this is irrelevant

Reply:
My editor was misconfigured. Done. Thanks